odoo/addons/base/tests/test_mail.py

```diff
@@ -366,6 +368,30 @@ def test_cid_with_at(self):
     #     for ext in test_mail_examples.MSOFFICE_1_OUT:
     #         self.assertNotIn(ext, new_html)
 
+    def test_multi_quote_detection(self):
+        doc = """
+            <div>
+                <div>
+                    My name is Green
+                </div>
+                <div> Hello </div>
+                --</br>Signature Hello </span>
+            </div>
+        """
```

Comment:
Your HTML here isn't legal you have a span closing tag without an opening one and, less important but still, it's `<br/>` not `</br>`.
Unless you intended the HTML to not be correct in the first place?

Reply:
Hello @thjo-odoo my bad html isn't legal. My intention wasn't to make the html illegal (I was playing around with too many examples and seems to have made changes in the block, that should have been reverted). I have changed the html block with more appropriate html block. You can have a look.

Thanks for the review, have a nice day !